graphql_api/types.py

```diff
@@ -682,3 +682,13 @@ class ZeroShotNRecords(graphene.ObjectType):
 class ZeroShotNRecordsWrapper(graphene.ObjectType):
     duration = graphene.Float()
     records = graphene.List(ZeroShotNRecords)
+
+
+class ModelProviderInfoResult(graphene.ObjectType):
+    name = graphene.String()
+    revision = graphene.String()
+    link = graphene.String()
+    date = graphene.Float()
```

Comment:
there is a graphene.Date object, why wasn't this used?

Reply:
changed